packages/origin-backend/migrations/initial.sql

```diff
@@ -0,0 +1,245 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp" WITH SCHEMA public;
```

Comment:
If this is autogenerated and should not be edited, please add a warning `// Autogenerated - DO NOT MODIFY`.

Reply:
Added warning message in the README.md file.

Reply:
Otherwise we would have to add this message in every new migration.